documentation/docs/release-notes/0.42.0.md

```diff
@@ -13,7 +13,7 @@
 
 They are incorporated in the province of Alberta and registered as a Money Service Business (MSB), making them subject to the Proceeds of Crime (Money Laundering) and Terrorist Financing Act (PCMLTFA) and applicable regulatory framework of the Financial Transactions and Reports and Analysis Centre of Canada (FINTRAC).
 
-Read more in our documentation: [How to use NDAX connector](/connectors/ndax)
+Read more in our documentation: [How to use NDAX connector](/spot-connectors/ndax)
```

Comment:
I will make this fix to the old docs repo, but this fix shouldn't be here in this one, since we changed the folder structure in the new docs so that all connectors reside in the `/connectors` directory